features/networking/service.feature

```diff
@@ -153,3 +153,438 @@ Feature: Service related networking scenarios
     Then the step should succeed
     And the output should contain:
       | 8080 |
+
+  # @author weliang@redhat.com
+  # @case_id OCP-24668
+  @admin
+  Scenario: externalIP defined in service but no sepc.externalIP defined	
+    Given I have a project 
+    And evaluation of `project.name` is stored in the :proj1 clipboard
+    
+    # Create a service with a externalIP in :proj1
+    When I run the :create client command with:
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/externalip_service1.json | 
+      | n | <%= cb.proj1 %>                                                                                        |
+    Then the output should contain "externalIPs have been disabled" 
+    
+    
+  # @author weliang@redhat.com
+  # @case_id OCP-24669
+  @admin
+  Scenario: externalIP defined in service with set ExternalIP in allowedCIDRs	 
+    # Create additional network through CNO
+    When I run the :patch admin command with:
+      | resource      | networks.config.openshift.io                                        |
+      | resource_name | cluster                                                             |
+      | p             | {"spec":{"externalIP":{"policy":{"allowedCIDRs":["22.2.2.0/24"]}}}} |
+      | type          | merge                                                               | 
+    Then the step should succeed
+    Then I wait up 10 seconds for the steps to pass:
+    """
+    When I run the :get admin command with:
+      | resource      | networks.config.openshift.io                 |
+      | resource_name | cluster                                      |
+      | template      | {{{{.spec.externalIP.policy.allowedCIDRs}}}} |
+      | o             | yaml                                         |
+    And the output should contain "22.2.2.0"
+    """
```

Comment:
I would suggest to replace all steps above with:
```
/^(?:(as admin) )?I successfully#{OPT_WORD} patch resource "(.*)\/(.*)" with:$/
```
i.e.
```
Given I successfully merge patch resource networks.config.openshift.io/cluster with:
  | {"spec":{"externalIP":{"policy":{"allowedCIDRs":["22.2.2.0/24"]}}}} |
```

Reply:
btw this seems to be changing configuration of a global cluster resource. This can easily interfere with other scenarios running simultaneously on this cluster. Please correct me if I'm wrong.

So you need to add tag `@destructive` to scenario as well `destructive` in Polarion. Updating polarion with the script will set that tag automatically provided it is set here.

Please see `doc/admin_features.adoc` about clean-up that is necessary in addition to setting the tags. Let me know if something is not clear around the whole `destructive` testing topic.

Reply:
Please update all scenarios according to these guidelines (where applicable). I didn't check all of them but I see no point in putting similar comments in multiple places.

Thank you!

Reply:
Update to the new to download file and replace paths
Add @@destructive tag in five test cases